.github/workflows/main.yaml

```diff
@@ -11,8 +21,17 @@ jobs:
         with:
           toolchain: nightly
           target: wasm32-wasi
+          components: clippy, rustfmt
           default: true
 
+      - uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          args: --all-features
+
+      - name: format source code
+        run: cargo fmt -- --check
```

Comment:
can we use `actions-rs/cargo`? or, it seems like `actions-rs` should have something for clippy.  not required - just seems like it would have nice integration features?

Reply:
I'm not sure I follow. I'm using the action-rs/clippy-check action already.

Reply:
Sorry - I mean rather than doing this:

```
- name: format source code
  run: cargo fmt -- --check
```
is there an advantage to doing this?
```
- uses: actions-rs/cargo@v1
        with:
          command: fmt -- --check
```